### PR TITLE
Changelog for 4.3.1

### PR DESCRIPTION
**Pull request by bgavrilMS**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,13 @@
+4.3.1
+=============
+Bug Fixes:
+
+- **.WithCertificate with /common audience scenario is broken**. Confidential Client authorization flow and OBO were not able to use certificates with the common authority set. More details [here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/891)
```
Comment (jmprieur) [inline changelog.txt]: is => was

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+- **Do not strip the port from authority Uri**. When passing your own authority uri which includes a port, MSAL used to strip out the port from the URI, making the authority unreachable. More details [here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1292)
```
Comment (jmprieur) [inline changelog.txt]: MSAL.NET no longer strips the port from the authority URI.

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+- **Fix a crash on Android when Chrome isn't installed on the device**. Exception is NameNotFoundException: com.android.chrome. More details [here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1334)
```
Comment (jmprieur) [inline changelog.txt]: Fix => Fixed. Exception was

File: changelog.txt [hunk 4]
```diff
@@ -1,3 +1,13 @@
...
+- **ConfidentialClient built from options won't allow certificates**. When building a confidential client from options, MSAL was forcing developers to use a secret. More details [here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1332)
```
Comment (jmprieur) [inline changelog.txt]: won't => did not?

File: src/Directory.Build.props [hunk 5]
```diff
@@ -5,10 +5,10 @@
     <TargetFrameworkNetDesktop>net45</TargetFrameworkNetDesktop>
     <TargetFrameworkNetStandard>netstandard1.3</TargetFrameworkNetStandard>
     <TargetFrameworkNetCore>netcoreapp2.1</TargetFrameworkNetCore>
-    <TargetFrameworkUap>uap10.0</TargetFrameworkUap>
+    <!--<TargetFrameworkUap>uap10.0</TargetFrameworkUap>
```
Comment (jmprieur) [inline src/Directory.Build.props]: What is it @bgavrilMS ?

File: changelog.txt (hunk 4 above)
Comment (bgavrilMS) [reply changelog.txt]: didn't :)

File: src/Directory.Build.props (hunk 5 above)
Comment (bgavrilMS) [reply src/Directory.Build.props]: Reverting..

**Pull request merged**